verify/cmd/runner.go

```diff
@@ -87,6 +87,10 @@ func main() {
 					errs = append(errs, "**Your PR has an issue number in the title.**\n\nThe title should just be descriptive.\nIssue numbers belong in the PR body as either `Fixes #XYZ` (if it closes the issue or PR), or something like `Related to #XYZ` (if it's just related).")
 				}
 
+				if regexp.MustCompile(`(?s)<!--.*?-->`).MatchString(pr.GetBody()) {
```

Comment:
this'll get comments inside code blocks, and other stuff too.  At the very least we need to check https://github.com/russross/blackfriday/blob/v2.0.1/html.go#L68.

The "more correct" solution is probably to parse the markdown with blackfriday & look for html nodes that are comment nodes, I think.

Reply:
+1 for the "more correct" solution